kubernetes/clean-up-jobs.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash -e
 
-kubectl delete job reset-database   || true
-kubectl delete job migrate-database || true
+kubectl delete job reset-database-control   || true
+kubectl delete job reset-database-storage   || true
+kubectl delete job migrate-database-control || true
+kubectl delete job migrate-database-storage || true
```

Comment:
You changed `reset-database` to `reset-database-control` and `reset-database-storage`. Similarly for `migrate-database`. `reset-control-database` and `reset-storage-database` would be more natural and I assumed that you would do it that way.

Anyway, it's not a big deal. I've accepted the pull request anyway because your names are acceptable too. I would change them but I'll leave it up to you.

Reply:
Done.